pkg/plugin/plugin.go

```diff
@@ -14,3 +17,13 @@ type Command struct {
 	Path        string `toml:"path" json:"path"`
 	Description string `toml:"description" json:"description"`
 }
+
+// Dir gives the path to the plugin's env directory on the filesystem.
+func (p *Plugin) Dir() string {
+	return p.dir
+}
+
+// CompletionDir returns the absolute path to the directory holding the plugin's completion script files
+func (p *Plugin) CompletionDir() string {
+	return filepath.Join(p.Dir(), "completion")
```

Comment:
The completion command requires `bash` as an argument, this gives us the flexibility to support other shell completion in the future. Can we reflect this here too? If we were to support zsh in the future for example where would we put the scripts.

Reply:
Should we go with `completion/<shell>` or something more like `completion.<shell>/`?

Reply:
I like `completion/<shell>` :+1:

Reply:
This was done in the script instead since it didn't make much sense to add some kind of shell flag to `dcos plugin list --completion-dirs` and each shell will know which shell it is.